src/services/userServices.js

```diff
@@ -0,0 +1,21 @@
+const angular = require('angular');
+
+angular.module('app-bootstrap').factory('logIn', [
+  function login(email,password) {
+    $http({
+      url: 'https://books-training-rails.herokuapp.com/api/v1/users/sign_in',
+      method: "POST",
+      data: { 'email' : email,
+              'password' : password}
+    })
+    .then(function(response) {
+      console.log(response);
+    }, 
```

Comment:
We can use an arrow function here to reduce the code
```suggestion
    .then(response => {
      console.log(response);
    }, 
```